brigade/core/configuration.py

```diff
@@ -24,6 +24,40 @@
         'default': os.path.join(os.path.expanduser("~"), ".ssh", "config"),
         'default_doc': '~/.ssh/config'
     },
+    'logging_dictConfig': {
+        'description': 'Configuration dictionary schema supported by the logging subsystem. '
+                       'Overrides rest of logging_* parameters.',
+        'type': 'dict',
```

Comment:
When there is a configuration object with the type of "dict" should we have some restriction so you aren't able to set that value using an environment variable?

Reply:
Yeah, we should. But I don't think it's super important so maybe we can create an issue and fix it later with calm.